framer/Animation.coffee

```diff
@@ -294,8 +296,14 @@ class exports.Animation extends BaseClass
 		@_target[key] = Color.mix(@_stateA[key], @_stateB[key], value, false, @options.colorModel)
 
 	_updateGradientValue: (key, value) =>
+		if not @_stateB[key] and value is 1
+			@_target[key] = @_stateB[key]
```

Comment:
Yes, because we override `@_stateB[key]` to have “something” to animate to, we make sure that we do set the original value when the animation finishes.